product_docs/docs/pem/9/registering_agent.mdx

```diff
@@ -48,23 +50,23 @@ set PEM_SERVER_PASSWORD=edb
 | `--pem-server`                  | The IP address of the PEM backend database server. This parameter is required.                                                                                                                                                                                                    |
 | `--pem-port`                    | The port of the PEM backend database server. The default value is `5432`.                                                                                                                                                                                                         |
-| `--pem-user`                    | The name of a database user with the `pem_admin` role and the `rolcreaterole` flag set (or a superuser) on the PEM backend database server. This user will be used to connect to the PEM server to perform agent registration.  This parameter is required.                                                                                                                                                         |
-| `--pem-agent-user`              | The name of a database user on the PEM backend database server. After registration, the agent will use this user to open connections to the PEM database server to write probe data, evaluate alerts, etc. This parameter is optional. If omitted, the agent will connect using a new user created during registration named `agent<N>` where `<N>` is the agent ID. Note that this user is always created and even if you specify a `pem-agent-user` the agent will use `SET ROLE agent<N>` to switch roles after the connection is made. |
-| `--pem-ssl-mode`                | The [SSL mode](https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-PROTECTION) to be used by the PEM agent user (see above). The possible values are `prefer`, `require`,`disable`,`verify-CA`, and `verify-full`. The default value is `require`.                                                                                                              |
-| `--cert-path`                   | The complete path to a directory in which certificates will be stored. If you don't provide a path, certificates are created in `~/.pem` on Linux and `%APPDATA%/pem` on Windows.                                                                                                     |
+| `--pem-user`                    | The name of a database user with the pem_admin role and the `rolcreaterole` flag set on the PEM backend database server, or a superuser. This user will be used to connect to the PEM server to perform agent registration. This parameter is required.                                                                                                                                                         |
```

Comment:
Please check this revision. The parenthetical made it harder to read and I wasn't sure if this is what was meant. Would two sentences be clearer?

Reply:
@sonotley Would mind taking a look at this edit so we can merge this one if all is well? Thanks!